yarn.lock

```diff
@@ -566,6 +566,11 @@
     pouchdb-collections "^1.0.1"
     tiny-queue "^0.2.1"
 
+"@okta/configuration-validation@^0.1.1":
+  version "0.1.1"
```

Comment:
Wonder, should this be 0.2.0?

Reply:
It should...and I have no idea why it's not updating.